library/objective-c/EnvoyConfiguration.m

```diff
@@ -23,12 +25,15 @@ - (instancetype)initWithDomain:(NSString *)domain
 - (nullable NSString *)resolveTemplate:(NSString *)templateYAML {
   NSDictionary<NSString *, NSString *> *templateKeysToValues = @{
     @"domain" : self.domain,
+    @"stats_domain" : self.statsDomain,
     @"connect_timeout_seconds" :
         [NSString stringWithFormat:@"%lu", (unsigned long)self.connectTimeoutSeconds],
     @"dns_refresh_rate_seconds" :
         [NSString stringWithFormat:@"%lu", (unsigned long)self.dnsRefreshSeconds],
     @"stats_flush_interval_seconds" :
-        [NSString stringWithFormat:@"%lu", (unsigned long)self.statsFlushSeconds]
+        [NSString stringWithFormat:@"%lu", (unsigned long)self.statsFlushSeconds],
+    @"device_os" : @"iOS"
```

Comment:
Android uses `android`. Seems like we should be consistent and either do `Android`/`iOS` or `android`/`ios`

Reply:
went for the former.